common/app/views/commercial/cards/itemCard.scala.html

```diff
@@ -16,19 +16,19 @@
 <div class="@CssClassBuilder.linkFromStandardCard(item, optAdvertClassNames, optClassNames, useCardBranding)">
     <a href="@item.targetUrl"
        data-link-name="@omnitureId"
-       class="advert">
+       class="advert__container">
```

Comment:
I thought `.advert__container` would be used for the surrounding `DIV` 😅  The idea is that everything that happens inside an `.advert` (its layout and sizes and stuff) does not need to apply to the surrounding `DIV`.